experimental/packages/otlp-transformer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/otlp-transformer",
-  "private": true,
+  "private": false,
   "publishConfig": {
     "access": "restricted"
```

Comment:
Yes. If we were to release after this PR has merged, then this package would need to be public.

Reply:
The `"private": true` above is changed, I expect it would go together with the "access". Any reason to do it later? After the PR is merged, public exporter packages would depend on this package which means it should be in npm if I understand the plan correctly

Reply:
Oh, yes you're correct. I forgot to change that to `"public"`. Fixed in [4507c52](https://github.com/open-telemetry/opentelemetry-js/pull/2929/commits/4507c5211573ed3e30228f5b73c8daa7ff512bf6). Thanks for catching this! :slightly_smiling_face: